src/include/daos/dtx.h

```diff
@@ -35,7 +35,7 @@
 #define DTX_THRESHOLD_COUNT		(1 << 9)
 
 /* The time (in second) threshould for batched DTX commit. */
```

Comment:
s/threshould/threshold

Reply:
> I don't see anything in this patch that matches the 2nd part of the description

It is "TODO:" ?

Reply:
The 2nd part is already implemented, this patch only shrink this value.

Reply:
Right, the 1 and 2 in commit comment are TODO.

Reply:
ah, sorry, I'm blind